src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int MAX_SIZE = 100;
+    private static Task[] userTasks = new Task[MAX_SIZE];
+    private static int userTasksCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userInput;
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("Hello! I'm Duke" + System.lineSeparator() + "What can I do for you?");
+
+        userInput = in.nextLine();
+
+        while (!userInput.equals("bye")) {
+            if (userInput.equals("list")) {
+                listUserTasks(userTasksCount);
+            } else if (userInput.contains("done")) {
+                int taskNumber = Integer.parseInt(userInput.substring(5));
+                markTaskAsDone(taskNumber);
+            } else {
+                addUserTask(userInput);
+            }
+            userInput = in.nextLine();
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    public static void addUserTask(String task) {
+        if (task.startsWith("todo")) {
+            task = task.replace("todo", " ").trim();
+            userTasks[userTasksCount] = new Todo(task);
+        } else if (task.startsWith("deadline")) {
+            task = task.replace("deadline", " ").trim();
+            String[] taskSplit = task.split(" /by ");
+            userTasks[userTasksCount] = new Deadline(taskSplit[0], taskSplit[1]);
+        } else if (task.startsWith("event")) {
+            task = task.replace("event", " ").trim();
+            String[] taskSplit = task.split(" /at ");
+            userTasks[userTasksCount] = new Event(taskSplit[0], taskSplit[1]);
+        } else {
+            return;
+        }
```

Comment:
Should command words like "todo", "deadline", "event" be defined in an enumeration?